.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: daily
```

Comment:
include updating of github actions too

Reply:
Thanks, I included also the open pull request limit to 10